src/tests/electron/tests/android-setup-prompt-connect-to-device.test.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    leftFooterButtonAutomationId,
+    rightFooterButtonAutomationId,
+} from 'electron/views/device-connect-view/components/android-setup/android-setup-step-layout';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { createApplication } from 'tests/electron/common/create-application';
+import { scanForAccessibilityIssues } from 'tests/electron/common/scan-for-accessibility-issues';
+import { AndroidSetupViewController } from 'tests/electron/common/view-controllers/android-setup-view-controller';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import {
+    commonAdbConfigs,
+    setupMockAdb,
+    simulateNoDevices,
+} from '../../miscellaneous/mock-adb/setup-mock-adb';
+
+describe('Android setup - prompt-connect-to-device ', () => {
+    const defaultDeviceConfig = commonAdbConfigs['multiple-devices'];
+    const detectButtonId = 'detect-device';
```

Comment:
can we import `detectDeviceAutomationId` from `prompt-connect-to-device-step.tsx` instead of redefining it here?

Reply:
Fixed